controllers/comment-controller.js

```diff
@@ -8,3 +8,13 @@ exports.getCommentsByArticle =( request, response, next) => {
       next(err)
   })
     }
+
+    exports.postCommentToArticle =(request, response, next) =>{
+      const articleID = request.params.article_id
+      const postInfo = request.body
+      addCommentToArticle(articleID, postInfo).then((result)=>{
+        response.status(200).send(result);
```

Comment:
would be good to send this back on an object and name as something representative such as {comment}